### PR TITLE
Update context aware documentation

### PR DESCRIPTION
**Pull request by flavio**
Cover a change introduced by the next release of Kubewarden.

This is related with https://github.com/kubewarden/kubewarden-controller/issues/575

This is the direct link to the new section: https://deploy-preview-355--silly-bunny-8cedd0.netlify.app/next/writing-policies/spec/context-aware-policies#constraints

### CODE REVIEW

File: docs/reference/spec/05-context-aware-policies.md [hunk 1]
```diff
@@ -34,6 +29,53 @@ That means some information might be stale or missing.
...
+
+Kubewarden policies can get a specific resource defined inside of the cluster. For example, they can make a query like
+"get the Pod named `psql-0` defined inside of the `db` namespace".
+
+By default this query fetches the object and stores it inside of an in-memory cache. The cache entry is wiped after 5 seconds.
+During this time window, the cached data is served to policies.
+
+The policy author can also decide to make a direct query, one that skips the cache entirely. In this way, fresh data is always
+server. This however can cause more load on the Kubernetes API server (depending on how frequently the policy is triggered)
+and introduces more latency when evaluating an admission request.
```
Comment (jhkrug) [inline docs/reference/spec/05-context-aware-policies.md:75]:
```suggestion
The policy author can also decide to make a direct query, one that skips the cache entirely. In this way, fresh data is always
served. This however can cause more load on the Kubernetes API server (depending on how frequently the policy is triggered)
and introduces more latency when evaluating an admission request.
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
lgtm. one tiny typo.

**Comment by flavio** [comment]
@jhkrug thanks for the fix. I've merged it

**Review by viccuad — APPROVED** [review]
LGTM!

**Pull request merged**